powerhome/playbook
# Make Button Kit Compatible with Turbo/Hotwire Features in Rails 7

#### Screens

<img width="1680" alt="Screen Shot 2022-05-27 at 12 01 10 PM" src="https://user-images.githubusercontent.com/38359249/170736522-6a20180e-8b8b-4ba6-bc1d-cc9ca509d6a2.png">

#### Breaking Changes

None, this change makes it so `target="_self"` is not added to link variants of the button kit. This is the default behavior for `a` tag, so removing this will have no affect on existing buttons. Specifying `new_window: true` in the props continues to add `target="_blank"`.

#### Runway Ticket URL

[SHD-41](https://nitro.powerhrg.com/runway/backlog_items/SHD-41)

#### How to test this

Go to Rails version of button kit, inspect HTML in "Button Links" section and verify the new window version has the `target="_blank"` attribute added, but that the other versions do not have a `target` attribute added.

#### Checklist:
